Fix a missing header include, required for S_ISVTX et al.

N.B. This change is in the upstream CVS HEAD, but is not in the latest
released tarball. When the next release tarball is made available, this
patch can be removed.

Signed-off-by: Simon Dawson <spdawson@gmail.com>

diff -Nurp a/minidlna.c b/minidlna.c
--- a/minidlna.c	2012-05-08 01:10:42.000000000 +0100
+++ b/minidlna.c	2013-05-04 21:28:35.372717737 +0100
@@ -57,6 +57,7 @@
 #include <arpa/inet.h>
 #include <fcntl.h>
 #include <sys/file.h>
+#include <sys/stat.h>
 #include <sys/time.h>
 #include <time.h>
 #include <signal.h>
